HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -757,6 +803,112 @@ def self.set_vacancy_periods(schedule, sch_name, vacancy_periods, year)
     end
   end
 
+  def self.set_power_outage_periods(schedule, sch_name, power_outage_periods, year)
+    return if power_outage_periods.nil?
+
+    # Add power outage rule(s), will override previous rules
+    power_outage_periods.each_with_index do |op, i|
+      value = 0.0
+
+      # Temperature of tank < 2C indicates of possibility of freeze.
+      value = 2.0 if sch_name.include?('WH Setpoint Temp') || sch_name.include?("#{Constants.ObjectNameWaterHeater} Setpoint")
+
+      next if sch_name.include?(Constants.ObjectNameNaturalVentilation) && op.natvent.nil?
+
+      value = 1.0 if sch_name.include?(Constants.ObjectNameNaturalVentilation) && op.natvent
```

Comment:
Set to 1 (always on) if natural ventilation schedule and power outage natvent indicates "always available during out period".